rdr_service/resource/schemas/genomics.py

```diff
@@ -177,6 +216,7 @@ class GenomicManifestFileSchema(Schema):
     rdr_processing_complete_date = fields.DateTime()
     ignore_flag = fields.Int16()
     file_name = fields.String(validate=validate.Length(max=255))
+    manifest_type_id_str = fields.String(validate=validate.Length(max=64))
```

Comment:
Maybe this isn't the convention for the genomics tables, but there is an enum class for these manifest types (`rdr_service/genomic_enums.py GenomicManifestTypes` .  Do we want to have the `manifest_type` / `manifest_type_id` paired fields like we do for other PDR schemas?

Reply:
Good catch, I didn't pay attention to this field.  The RDR devs have added an enum string field for the 'manifest_type_id' field, but we already handled having both the id and str fields previously.  I have removed the new field as it is not needed, in my next commit.

Reply:
Turns out I was adding a few unneeded enum string fields.  I should have looked harder at those new fields.  I'm glad you caught this.